### DISCUSSION TITLE
`Bllossom/llama-3-Korean-Bllossom-70B-gguf-Q4_K_M`모델을 한번더 양자화

### DISCUSSION THREAD

**Discussion by DDUKDAE** [boostcampaitech7/level2-nlp-generationfornlp-nlp-07-lv3#7] [Ideas]
`Bllossom/llama-3-Korean-Bllossom-70B-gguf-Q4_K_M`
이 모델은 llama ccp를 이용해서 양자화한 모델입니다
해당 허깅페이스 설명으로는 gpu 메모리 45g가 필요하다고 하는데요
따라서 모델을 조금더 양자화 시키면 성능이 좋게 나올 수도 있지 않을까 합니다

해당 모델은 4bit로 양자화된 상태인 것 같습니다
![image](https://github.com/user-attachments/assets/b4a2827e-962c-41be-b284-2048b15f45db)

그런데 다른 분이 만든 모델이 하나 더 있습니다
`legenduck/llama-3-Korean-Bllossom-70B-GGUF`
이건 2bit까지 양자화를 한 것 같더라구요
![image](https://github.com/user-attachments/assets/fc11a3a1-92b6-4982-a6cb-1de4f4042606)

한번 시도해볼만은 한것 같은데
llamaccp를 서버에 설치해본 경험이 없어서
하려면 오래 걸릴 것 같아 의견을 물어보고 싶습니다

**Comment by hskhyl** [comment]
저는 12.8B 짜리 8비트로 양자화 해서 했을때 그다지 성능이 안 좋았어서..!
성능 향상을 기대하기 보다는 좀 큰 빌리언 모델들을 양자화 했을 때 성능 자체가 얼마정도 찍히는지 확인하는 용도로는 좋은 것 같습니다.
